mw-webapp/src/component/errorBoundary/ErrorBoundary.tsx

```diff
@@ -0,0 +1,32 @@
+import React, {Component} from "react";
+import {Page404} from "src/pages/page404/Page404";
+
+interface Props {
+  children: React.ReactNode;
+}
+
+interface State {
+  hasError: boolean;
+}
+
+class ErrorBoundary extends Component<Props, State> {
```

Comment:
let's use functional component. Investigate does possible for error boundaries for react 18

Reply:
Done.